README.md

```diff
@@ -34,16 +34,6 @@ Before you begin, make sure you have the following prerequisites:
 
 - Git: Install Git on your system if you haven't already. You can find installation instructions at [https://git-scm.com/book/en/v2/Getting-Started-Installing-Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
 
-- zkSync CLI (only if you want to run a localnet): Install the zkSync CLI on your system
```

Comment:
I added installing local-setup further down, but maybe I should add it here too?

Reply:
It could be a good idea. It is the same for me.